src/main/resources/templates/dataresource.html

```diff
@@ -0,0 +1,184 @@
+		<object th:include="fragments/header :: header" th:remove="tag" />
+	</head>
+	<body class="mybg loginbackground">
+	<div th:replace="fragments/dashboard_navigation :: dashboardnav">&nbsp;</div>
+	<div class="container">
+		<p>&nbsp;</p>
+					<p>&nbsp;</p>
+					<p>&nbsp;</p>
+					<p>&nbsp;</p>
+					<!--here make changes-->
+					<!--<div class="tab-content col-sm-10 dashpanel_admin">-->
+					<div class="row dashpanel col-sm-12">
+						<!--<div class="tab-content col-sm-10">-->
+						<h2 class="team-heading team-heading-current-team" th:text="#{data.heading.current}">text heading from .properties</h2>
+
+						<ul class="topnav col-sm-2">
+							<li>
+								<a href="#">Data Category</a>
+								<ul>
+									<li class="active"><a data-toggle="tab" href="#vul">Malware Data</a></li>
+									<li><a data-toggle="tab" href="#bigdata">Application Layer Security Data</a></li>
+									<li><a data-toggle="tab" href="#bigdata">DNS Data</a></li>
+									<li><a data-toggle="tab" href="#bigdata">IDS and Firewall Data</a></li>
+									<li><a data-toggle="tab" href="#bigdata">IP Packet Headers</a></li>
+
+
+						<!--<ul>-->
+							<!--<li><a href="#">/a></li>-->
+							<!--<li><a href="#"></a></li>-->
+							<!--<li><a href="#"></a></li>-->
+						<!--</ul>-->
+				</ul>
+			</li>
+			<li><a href="#">Year Created</a>
+				<ul>
+					<li><a href="#">2016</a></li>
+					<li><a href="#">2015</a></li>
+					<li><a href="#">2014</a></li>
+
+				</ul>
+			</li>
+
+		</ul>
+
+		<div class="col-sm-9">
+			<div id="vul" class="tab-panel fade in active">
+				<h3>Malware Data</h3>
+					<!--<div class="form-inline left">-->
+				<!--&lt;!&ndash;<input type="search" class="filterTable" placeholder="Search..."/>&ndash;&gt;-->
+				<!--<button type="button" class="form-control"><i class="fa fa-search"></i></button>-->
+				<!--</div>-->
+
+
+				<!--<div class="input-group">-->
+				<!--<input type="search" class="form-control" placeholder="Search...">-->
+					<!--<span class="input-group-btn">-->
+    				<!--<button class="btn btn-default" button type="button">Go!</button>-->
+  					<!--</span>-->
+				<!--</div>-->
+				<div>
+					<table id="SUETable" class="table">
+					<tr>
+						<th></th>
+						<th width="25%">Data Name</th>
+						<th width="30%">Description</th>
+						<th width="15%">Release Date</th>
+						<th width="30%">Action</th>
+					</tr>
+					<tr>
+						<td>1</td>
+						<td><a href="#" data-toggle="modal" data-target="#myModal">ITDK 2015-08</a></td>
+						<div class="container">
+							<div id="myModal" class="modal fade admin-modal" role="dialog">
+								<div class="modal-dialog">
+									<!--Modal Content-->
+									<div class="modal-content">
+										<div class="modal-header">
+											<button type="button" class="close" data-dismiss="modal">&times;</button>
+											<h4 class="modal-title">ITDK 2015-08 </h4>
+										</div>
+										<div class="modal-body">
+											<p><b>Details</b>
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this <b> tag by <strong>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)